### PR TITLE
Postfix conditionals

### PR DESCRIPTION
**Pull request by osuushi**

### CODE REVIEW

File: babel-plugins/postfix-if.js [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+    expressionInspector,
+  ];
+
+  return {
+    visitor: {
+      CallExpression: function (path) {
+        var callee = path.node.callee;
+
+        if (callee.type !== 'Identifier') return;
+        if (!calleePattern.test(callee.name)) return;
```
Comment (benbernard) [inline babel-plugins/postfix-if.js]: isn't this expected to be exactly either `If` or `Unless` would seem better to me to not use the regex, but just an array to compare with

**Comment by benbernard** [comment]
looks good, one comment.

We could remove the jscs rules, we don't sue them anymore, right? I think we switched everything to eslint

File: babel-plugins/postfix-if.js (hunk 1 above)
Comment (osuushi) [inline babel-plugins/postfix-if.js]: Sure.

**Comment by benbernard** [comment]
What do we want to do with this PR?

**Comment by benbernard** [comment]
we've decided this is a bridge too far

**Pull request closed without merging**